main.js

```diff
@@ -90,6 +89,48 @@ const renderMainWindow = () => {
         });
     }
   })();
+  
+  //read file and show it in app
+  ipcMain.handle('request-load-text', () => {
+    return fs.readFileSync(filePath, "utf8");
+  });
+
+  function getMasterPassword() {
+    return JSON.parse(fs.readFileSync(passwordFilePath, 'utf-8'));
+  }
+  
+  ipcMain.on('save-and-encrypt-text', async (event, text) => {
+    const userFilesDir = path.dirname(process.env.USER_FILE_PATH);
+    const filePath = path.resolve(process.env.USER_FILE_PATH);
+    const encryptedFilePath = path.join(userFilesDir, 'encrypted.txt');
+  
+    try {
+      fs.writeFileSync(filePath, text, 'utf8');
+      console.info(`File saved at: ${filePath}`);
+  
+      const passwordData = getMasterPassword();
+      const { hashedPassword } = passwordData;
+  
+      const encryptedText = sjcl.encrypt(hashedPassword, text);
+  
+      fs.writeFileSync(encryptedFilePath, encryptedText, 'utf8');
+      console.info(`Encrypted file saved at: ${encryptedFilePath}`);
```

Comment:
it's more grammar right way `Encrypted file has been saved to ${encryptedFilePath}`
@RGamz am i right?
and I think, we have to add time log to this line for example
`Encrypted file has been saved to ${encryptedFilePath} at 2024-11-01 12:12:34` <- datetime string format
This log will be more useful 👆